File: app/src/test/java/ml/melun/mangaview/model/EpisodeLoadResultTest.java
package ml.melun.mangaview.model;

import static org.junit.Assert.assertEquals;

import java.util.Collections;

import ml.melun.mangaview.mangaview.Manga;
import org.junit.Test;

public class EpisodeLoadResultTest {
    @Test
    public void storesCodeAndEpisodes() {
        Manga episode = new Manga(7, "episode", "", 1);
        EpisodeLoadResult result = new EpisodeLoadResult(0, Collections.singletonList(episode));

        assertEquals(0, result.getResultCode());
        assertEquals(Collections.singletonList(episode), result.getEpisodes());
    }
}

File: app/src/test/java/ml/melun/mangaview/model/EpisodeLoadResultTest.kt
package ml.melun.mangaview.model

import ml.melun.mangaview.mangaview.Manga
import org.junit.Assert.assertEquals
import org.junit.Assert.assertTrue
import org.junit.Test

class EpisodeLoadResultTest {
    @Test
    fun storesCodeAndEpisodes() {
        val episodes = listOf(Manga(7, "episode", "", 1))
        val result = EpisodeLoadResult(0, episodes)

        assertEquals(0, result.resultCode)
        assertEquals(episodes, result.episodes)
    }
}
